app/helpers/Content/Guide.php

```diff
@@ -207,4 +217,29 @@ public function getSEOTitle()
             return $this->getTitle();
         }
     }
+
+    /**
+     * @return string
+     */
+    public static function fixDuplicateAnchorIDs($content, $anchors)
+    {
+        $anchorList = [];
+        foreach ($anchors as $anchor) {
+            $anchorList[$anchor['sub']][] = $anchor['parent'];
```

Comment:
Can a subsection have multiple parents? Basically wondering if we need to store an array of parents or just a single parent value.

Reply:
A subsection will only have one parent